### DISCUSSION TITLE
How do I disable an extension/securly on MAC?

### DISCUSSION THREAD

**Discussion by synapt0** [3kh0/ext-remover#526] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
Is there anything like the ext-remover that works on MAC?

**Comment by qDread-dev** [comment]
Not to familiar with mac, but couldn't you just install a new browser / browser profile?

**Comment by Aka-but-nice** [comment]
Point Blank would still work.

**Comment by funkymokey** [comment, marked as answer]
just dont have a mac (super simple fix)

**Reply by Aka-but-nice** [reply]
> just dont have a mac (super simple fix)

You have just one the noble peace prize, for being the most helpful person, also 3kh0 will have to make you colab

**Reply by Aka-but-nice** [reply]
> > just dont have a mac (super simple fix)
> 
> You have just one the noble peace prize, for being the most helpful person, also 3kh0 will have to make you collab

**Reply by Aka-but-nice** [reply]
> > > just dont have a mac (super simple fix)
> > 
> > 
 You have just won the noble peace prize, for being the most helpful person, also 3kh0 will have to make you collab

**Reply by obkduishinijf** [reply]
> > > > just dont have a mac (super simple fix)
> 
> You have just won the noble peace prize, for being the most helpful person, also 3kh0 will have to make you collab

bro's talking to his demons

**Comment by amelia-bartlett** [comment]
> Is there anything like the ext-remover that works on MAC?

if you have root (admin for you skids) on the mac, create a policy conflict
unfortunately, MacOS is the only system where I have no idea how to do this (its easy on Linux and Windows, but on MacOS idk)
A policy set on the machine will override policies from a google account
